mono/utils/mono-time.c

```diff
@@ -19,6 +20,18 @@
 
 #define MTICKS_PER_SEC (10 * 1000 * 1000)
 
+typedef enum _TimeConversionConstants
+{
+    tccSecondsToMillieSeconds       = 1000,         // 10^3
+    tccSecondsToMicroSeconds        = 1000000,      // 10^6
+    tccSecondsToNanoSeconds         = 1000000000,   // 10^9
+    tccMillieSecondsToMicroSeconds  = 1000,         // 10^3
+    tccMillieSecondsToNanoSeconds   = 1000000,      // 10^6
+    tccMicroSecondsToNanoSeconds    = 1000,         // 10^3
+    tccSecondsTo100NanoSeconds      = 10000000,     // 10^7
```

Comment:
Personally I think "100NanoSeconds" should be called "NtTime".

Reply:
`NtTime` implies implicit knowledge that it is 100ns, which is absolutely unclear to anybody who didn't do a lot of Windows dev.